server/@types/shared/models/ApprovedPremises.ts

```diff
@@ -2,11 +2,13 @@
 /* tslint:disable */
 /* eslint-disable */
 
+import type { LocalAuthorityArea } from './LocalAuthorityArea';
```

Comment:
I think these type updates should be in a different commit/PR

Reply:
Yeah, I dunno why those haven't come in a PR